src/dymaptic.GeoBlazor.Core/Components/Views/MapView.razor.cs

```diff
@@ -14,6 +14,13 @@
 
 namespace dymaptic.GeoBlazor.Core.Components.Views;
 
+/// <summary>
+/// Delegate Task that can be used asyncronously
+/// </summary>
+/// <param name="sender"></param>
```

Comment:
I see why you couldn't use Blazor's `EventCallback` for this event, since it's being listened for in a child component. However, a few things:
- I would move this to the bottom of the file, or to its own file
- Is the `sender` parameter necessary? Unless we're going to use this pattern outside of `MapView`, the sender is obvious and already known.
- Go ahead and delete xml comment lines for `param` and `returns` if they are obvious. Our docs site (docs.geoblazor.com) is partially generated from XML comments, so we don't want to leave empty ones around.